packages/vscode-docker-registries/src/clients/Common/CommonRegistryDataProvider.ts

```diff
@@ -55,7 +59,7 @@ export abstract class CommonRegistryDataProvider implements RegistryDataProvider
                 collapsibleState: vscode.TreeItemCollapsibleState.None,
                 contextValue: getContextValue(element, 'commontag'), // TODO
                 iconPath: new vscode.ThemeIcon('bookmark'),
-                description: element.createdAt?.toString() || '',
+                description: dayjs(element.createdAt?.toString() || undefined).fromNow() || '',

```

Comment:
Does dayjs not have a method for getting relative time from `Date`, only `string`?

Reply:
ohh yup you're right, it does take `Date`